app/models/post.rb

```diff
@@ -4,8 +4,13 @@ class Post < ApplicationRecord
   has_many :likes, dependent: :destroy
   after_save :increment_posts_counter
 
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }
+  validates :LikesCounter, numericality: { greater_than_or_equal_to: 0 }
+  validates :CommentsCounter, numericality: { greater_than_or_equal_to: 0 }
+
   def increment_posts_counter
-    user.increment!(:posts_counter)
+    user.increment!(:PostsCounter)
   end
 
   def recent_comments
```

Comment:
- [ ]  Good job with test coverage 💪, but since we are aiming for 100% test coverage, and it is also a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, "Add unit specs for all of your models' methods and validations", please add a unit tests for `increment_posts_counter` and `recent_comments` method.